src/addons/__tests__/ReactComponentWithPureRenderMixin-test.js

```diff
@@ -50,20 +50,20 @@ describe('ReactComponentWithPureRenderMixin', function() {
       getInitialState: function() {
         return {
           cut: false,
-          slices: 1,
+          slices: 1
```

Comment:
In this commit can we turn off the rule and not remove all the trailing commas (just to add them again a little later)? Also happy for us to add them everywhere before landing this PR if you'd rather do that.